python/lsst/ap/association/packageAlerts.py

```diff
@@ -0,0 +1,247 @@
+# This file is part of ap_association.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ("PackageAlertsConfig", "PackageAlertsTask")
+
+import os
+import tempfile
+
+import lsst.alert.packet as alertPack
+import lsst.geom as geom
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+from lsst.utils import getPackageDir
+
+
+"""Methods for packaging Apdb and Pipelines data into Avro alerts.
+"""
+
+
+class PackageAlertsConfig(pexConfig.Config):
+    """Config class for AssociationTask.
+    """
+    schemaFile = pexConfig.Field(
+        dtype=str,
+        doc="Location to write alerts to.",
+        default=os.path.join(getPackageDir("sample_avro_alert"),
+                             "schema/2/1/lsst.alert.avsc"),
+    )
+    cutoutSize = pexConfig.RangeField(
+        dtype=int,
+        min=0,
+        max=1000,
+        default=30,
+        doc="Dimension of the square image cutouts to package in the alert."
+    )
+    alertWriteLocation = pexConfig.Field(
+        dtype=str,
+        doc="Location to write alerts to.",
+        default=os.path.join(os.getcwd(), "alerts"),
+    )
```

Comment:
I'm a bit worried about putting yet more paths into our configs, since this effectively adds another command-line argument to `ap_pipe` (as in lsst/ap_verify#86). What's the long-term plan for these? Are they going to be Butler datasets?

Reply:
Long term, they should go direct to Kafka, without hitting persistent storage in the pipeline. Short term... better suggestions welcome! :-)